src/libc/expm1.c

```diff
@@ -1,8 +1,24 @@
 #include <math.h>
 
-float expm1f(float x)
+float expm1f(const float x)
```

Comment:
Why? That's not needed or in the standard.

Reply:
Adding const to parameters passed by value doesn't change the ABI. It was a micro optimization I added, although it probably doesn't change the compiler output